packages/launchpad/src/setup/ManualInstall.vue

```diff
@@ -35,6 +36,7 @@ import { computed } from 'vue'
 import CopyButton from '@cy/components/CopyButton.vue'
 import { gql } from '@urql/core'
 import type { ManualInstallFragment } from '../generated/graphql'
+import ExternalLink from '@packages/frontend-shared/src/gql-components/ExternalLink.vue'
```

Comment:
Do we need this import (since it's registered globally)?

I am not really a fan of global component registration in general, it's really hard to remember what's registered and what's not, but if we are using the global pattern I am not sure we should import the component in some places, but not others.

Reply:
Whoops, yeah the global import/registration should have been removed already, I'll take it out.